Assets/_BForBoss/_Core/Scripts/Enemies/FloatingTargetBehaviour.cs

```diff
@@ -6,18 +6,28 @@
 namespace BForBoss
 {
     [RequireComponent(typeof(AgentNavigationBehaviour))]
-    public class FloatingTargetBehaviour : EnemyBehaviour
+    public class FloatingTargetBehaviour : EnemyBehaviour, IKnockback
     {
         [SerializeField] private HealthbarViewBehaviour _healthbar;
+
+        [Header("Knockback Settings")]
+        [SerializeField] private float _knockbackRadiusModifier = 10f;
+        [SerializeField] private float _knockbackUpwardsForceModifier = 10f;
+        [SerializeField] private float _knockbackMaxDuration = 2f;
```

Comment:
Don't have to do it now, but if you feel like the `FloatingTargetBehaviour` is too bloated with different things, this could be abstract out into a separate knockbackBehaviour class component,  feel free to do so.

Reply:
Yeah I like your idea of having VFX trigger on knockback end and I think that will really start to bloat this. I'll leave that feature itself for later but pull this out for now